src/status_im2/contexts/chat/lightbox/style.cljs

```diff
@@ -77,8 +78,24 @@
    :align-items        :center
    :justify-content    :center})
 
-(def text-style
-  {:color             colors/white
-   :align-self        :center
-   :margin-horizontal 20
-   :margin-vertical   12})
+
+(defn background
+  [{:keys [overlay-opacity]} {:keys [overlay-z-index]}]
+  (reanimated/apply-animations-to-style
+   {:opacity overlay-opacity}
+   {:background-color colors/neutral-100-opa-70
+    :position         :absolute
+    :top              0
+    :bottom           0
+    :z-index          @overlay-z-index
```

Comment:
I think it's better in general if we keep style namespaces free of state as much as we can, i.e. `deref` the argument `overlay-z-index` before passing its value to `background`.